core/src/main/java/io/questdb/griffin/RecordToRowCopierUtils.java

```diff
@@ -236,13 +236,10 @@ public static RecordToRowCopier generateCopier(
                     break;
                 case ColumnType.IPv4:
                     asm.invokeInterface(rGetIPv4);
-                    switch (toColumnTypeTag) {
-                        case ColumnType.IPv4:
-                            asm.invokeInterface(wPutInt, 2);
-                            break;
-                        default:
-                            assert false;
-                            break;
+                    if (toColumnTypeTag == ColumnType.IPv4) {
```

Comment:
nit: could be simplified to
```java
assert toColumnTypeTag == ColumnType.IPv4;
asm.invokeInterface(wPutInt, 2);
```

Reply:
simplified in https://github.com/questdb/questdb/pull/3399/commits/5128f9e598617330b6ec25fdd85ff84326a154cc